tests/test_knowledge_panels.py

```diff
@@ -415,7 +420,7 @@ def test_data_quality_kp_with_all_tags(monkeypatch):
     }
 
 
-def test_last_edits_kp_with_one_facet_and_value(monkeypatch):
+async def test_last_edits_kp_with_one_facet_and_value(monkeypatch):
```

Comment:
Did you install an [async plugin](https://docs.pytest.org/en/latest/reference/plugin_list.html?highlight=async) for pytest ?

If yes you should add it to requirements.

Reply:
yes, pytest-asyncio, sorry I forget to add!